zhaquirks/tuya/ts011f_plug.py

```diff
@@ -1170,6 +1170,65 @@ class Plug_v3(EnchantedDevice):
     }
 
 
+class Plug_v3_noGP(EnchantedDevice):
+    """Tuya TS011F plug without GP."""
+
+    quirk_id = TUYA_PLUG_ONOFF
+
+    signature = {
+        MODELS_INFO: [
+            ("_TZ3000_cehuw1lw", "TS011F"),
+        ],
```

Comment:
To be honest, I think we can just match them to all `MODEL: "TS011F",` here, instead of only the specific model.
There seems to be a bunch of `TS011F` plugs.

Reply:
Maybe add your model in the docstring/comment for the class, so we have one example of a possible variant.